src/main/java/io/rocketpartners/cloud/action/rest/RestGetAction.java

```diff
@@ -267,8 +268,20 @@ else if (results.size() == limit && (foundRows < 0 || (offest + limit) < foundRo
             mapToColumns(collection, term);
          }
       }
+      Results results = null;
 
-      Results results = collection.getDb().select(collection.getTable(), terms);
+      if (collection == null)
+      {
+         Db db = api.findDb((String) Chain.peek().get("db"));
+         if (db == null)
+            throw new ApiException(SC.SC_400_BAD_REQUEST, "Unable to find collection");
+         results = db.select(null, terms);
+      }
+      else
+      {
+         results = collection.getDb().select(collection.getTable(), terms);
+      }
+      
```

Comment:
for specific queries without a collection in the request. Based on line 294 below, this appears to be a valid use case, like for SqlSuggestAction.